### DISCUSSION TITLE
Ice version && gcc version

### DISCUSSION THREAD

**Discussion by kexiongwang** [zeroc-ice/ice#2870] [Q&A]
wheather ice v 3.7.1 match gcc 9.4 ?

**Comment by pepone** [comment]
Ice works with a wide range of GCC versions, for a list of officially supported platforms see https://doc.zeroc.com/ice/3.7/release-notes/supported-platforms-for-ice-3-7-10

**Reply by kexiongwang** [reply]
Thanks，another question ：How can I achieve cross Ice master node subscription and publication directly through 0strom, i start 0strom at two  ice master nodes,with the same topic , but i cant publish or subscribe another master nodes topic , if i stop 0strom server at on node ,i can receive the other nodes topic .

**Reply by pepone** [reply]
See https://doc.zeroc.com/ice/3.7/ice-services/icestorm/highly-available-icestorm

And this demo: https://github.com/zeroc-ice/ice-demos/tree/3.7/cpp11/IceStorm/replicated2

**Reply by Fireshtorm1k** [reply]
> Thanks，another question ：How can I achieve cross Ice master node subscription and publication directly through 0strom, i start 0strom at two ice master nodes,with the same topic , but i cant publish or subscribe another master nodes topic , if i stop 0strom server at on node ,i can receive the other nodes topic .

I think you can mark pepone's reply as the solution, and close the task.
